mmv1/templates/terraform/update_encoder/filestore_instance.go.tmpl

```diff
@@ -0,0 +1,72 @@
+{{- if ne $.TargetVersionName "ga" }}
```

Comment:
I think we restrict any kind of API requests within encoders, and should have the logic moved to a `pre_update` template